player/tests.py

```diff
@@ -1,28 +1,41 @@
-from django.contrib.auth import get_user_model
 from .models import Player, BallGame
 from django.core.exceptions import ValidationError
+from django.contrib.auth.models import User
+from django.db import IntegrityError
 import pytest
 
 
 @pytest.mark.django_db
 class TestPlayerModel:
 
     @pytest.fixture
-    def user(self):
-        return get_user_model().objects.create_user(
-            username='testuser',
-            password='testpass'
-        )
-
-    @pytest.fixture
-    def saved_player(self, user):
-        player = Player.objects.create(
-            user=user,
+    def saved_player(self):
+        player = Player.create(
+            username="daniel",
+            password="password",
             birth_date='1990-01-01',
-            favorite_ball_game=BallGame.Soccer
-        )
+            favorite_ball_game=BallGame.Soccer)
         return player
 
+    def test_create_player_with_new_user(self):
+        player = Player.create(username='daniel',
+                               password='password',
+                               birth_date='1990-01-01',
+                               favorite_ball_game=BallGame.Soccer)
+        assert isinstance(player.user, User)
+        assert player.user.username == 'daniel'
+        assert player.user.check_password('password')
+
+    def test_create_player_with_existing_user(self):
```

Comment:
I think a better name will be `test_create_player_with_existing_username`, since you are not passing the `user` variable, and the creating is failed because in `Player.create` method we trying to create a new User with an existing username

Reply:
I agree.